src/styles/main.scss

```diff
@@ -1 +1,53 @@
-// put code here
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.stopwatch {
+  display: flex;
+  position: absolute;
+  width: 300px;
+  height: 300px;
+  border: 4px dashed black;
+  border-radius: 50%;
+  left: 50%;
+  top: 50%;
+  transform: translate(-50%, -50%);
+
+  &__min {
+    position: absolute;
+    height: 140px;
+    width: 5px;
+    background: black;
+    top: 50%;
+    left: calc(50% - 5px);
+    animation: spin 3600s linear infinite;
```

Comment:
The animation duration for the minute hand should be 3600s (60 minutes), but as per the requirement, it should rotate every 5s.